cmd/prcreator/main.go

```diff
@@ -28,7 +28,7 @@ func gatherOptions() (*options, error) {
 	flag.StringVar(&opts.prTitle, "pr-title", "", "The title of the PR to create")
 	flag.StringVar(&opts.prMessage, "pr-message", "", "The message of the PR to create")
 	flag.StringVar(&opts.gitCommitMessage, "git-message", "", "The git commit message of the PR to create. If not set, then its value will be composed of other flags")
-	flag.StringVar(&opts.prAssignee, "pr-assignee", "", "The assignee of the PR to create")
+	flag.StringVar(&opts.prAssignees, "pr-assignees", "", "Comma separated list of assignees for the PR to create")
```

Comment:
Changing the name here will break existing code before we  update the periodic and any other code that might be using it that we aren't aware of.  How about changing prAssignee to []string and the flag to `flag.StringArrayVar()` like https://github.com/openshift/ci-tools/blob/17e9c52f4342964afe8ff762c878435f0e3da992/pkg/jobrunaggregator/jobruntestcaseanalyzer/cmd.go#L134

then the code can pass in `-pr-assignee dgoodwin -pr-assignee  neisw  -pr-assignee DennisPeriquet` and the existing single case code should continue to work as well